assignments/objects.js

```diff
@@ -11,54 +11,122 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+  id: 0,
+  name: "Example",
+  email: "examples@you.edu",
+  gender: "F"
+};
 
 // Write your intern objects here:
 
-
-// ==== Challenge 2: Reading Object Data ==== 
+const Mitzi = {
+  id: 1,
+  email: "mmelloy0@psu.ed",
+  name: "Mitzi",
+  gender: "F"
+};
+
+const Kennan = {
+  id: 2,
+  email: "kdiben1@tinypic.com",
+  name: "Kennan",
+  gender: "M",
+  speak() {
+    return "Hello, my name is Kennan!";
+  }
+};
+
+const Keven = {
+  id: 3,
+  email: "kmummery2@wikimedia.org",
+  name: "Keven",
+  gender: "M"
+};
+
+const Gannie = {
+  id: 4,
+  email: "gmartinson3@illinois.edu",
+  name: "Gannie",
+  gender: "M"
+};
+
+const Antonietta = {
+  id: 5,
+  email: "adaine5@samsung.com",
+  name: "Antonietta",
+  gender: "F",
+  multiplyNums(num1, num2) {
+    return num1 * num2;
+  }
+};
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
+console.log(Mitzi.name);
 
 // Kennan's ID
+console.log(Kennan.id);
 
 // Keven's email
+console.log(Keven.email);
 
 // Gannie's name
+console.log(Gannie.name);
 
 // Antonietta's Gender
+console.log(Antonietta.gender);
 
-// ==== Challenge 3: Object Methods ==== 
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+console.log(Kennan.speak());
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
+console.log(Antonietta.multiplyNums(3, 4));
 
 // === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
 
-// ==== Stretch Challenge: Nested Objects and the this keyword ==== 
+// ==== Stretch Challenge: Nested Objects and the this keyword ====
```

Comment:
what do you keep doing to these comments that changing them?

Reply:
gotcha. I'm seeing a lot of other people doing it too so it's no big deal but I couldn't figure out what was happening